securitycenter/notification.rb

```diff
@@ -0,0 +1,152 @@
+# [START scc_create_notification_config]
+require "google/cloud/security_center"
+
+def create_notification_config org_id:, config_id:, pubsub_topic:
+  # org_id:       Your organization id. e.g. for organizations/123, this would
+  #               be 123.
+  # config_id:    Your notification config id. e.g. for config id
+  #               "organizations/123/notificationConfigs/my-config",
+  #               this would be "my-config".
+  # pubsub_topic: Pubsub topic that Notifications will be published to.
+  securitycenter = Google::Cloud::SecurityCenter.new
+
+  formatted_parent = securitycenter.organization_path org_id
+
+  notification_config = {
+    description:      "Sample config for Ruby",
+    pubsub_topic:     pubsub_topic,
+    streaming_config: { filter: 'state = "ACTIVE"' }
+  }
+
+  response = securitycenter.create_notification_config formatted_parent, config_id, notification_config
+  puts "Created notification config #{config_id}: #{response}. "
+end
+# [END scc_create_notification_config]
+
+# [START scc_update_notification_config]
```

Comment:
Can you move the region tags to inside the methods?

Reply:
I've migrated the region tags inside the methods now. PTAL!